.github/workflows/test-integrations-aws-lambda.yml

```diff
@@ -71,6 +71,8 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
+        env:
+          PIP_TRUSTED_HOST: "pypi.python.org pypi.org files.pythonhosted.org"
```

Comment:
We need this so that we don't get a certificate error in Python 3.5 GitHub actions.